Documentation/AEPTarget.md

```diff
@@ -99,6 +99,21 @@
     - [Objective C](#objc-visual-preview)
       - [Syntax](#objc-syntax-visual-preview)
       - [Example](#objc-example-visual-preview)
+- [Target Raw API reference](#target-raw-api-reference)
+- [executeRawRequest](#executerawrequest)
+    - [Swift](#swift-executerawrequest)
```

Comment:
Looks like these links to Swift, Objective C Syntax and Example doesn't work
And to me it feels like too many links right at top  
![Screenshot 2023-02-10 at 12 45 48 PM](https://user-images.githubusercontent.com/8909148/218194135-8205d877-1c25-4dc6-a620-2b05b89f467f.png)

Can we just keep the API links and remove all others ?

Reply:
I do see the links broken, probably an anchor tag is required. The original intent was to be able to point someone to a specific Swift or ObjC syntax/ example. I suppose we can clean this up. Ty.